backstop/tests/header.html

```diff
@@ -0,0 +1,969 @@
+<!DOCTYPE html>
+<html id="theme">
+  <head>
+    <title>Header | iTwinUI</title>
+    <link
+      rel="icon"
+      href="../assets/favicon.ico"
+    />
+    <link
+      rel="stylesheet"
+      href="../../lib/css/global.css"
+    />
+    <link
+      rel="stylesheet"
+      href="../../lib/css/button.css"
+    />
+    <link
+      rel="stylesheet"
+      href="../../lib/css/inputs.css"
+    />
+    <link
+      rel="stylesheet"
+      href="../../lib/css/user-icon.css"
+    />
+    <link
+      rel="stylesheet"
+      href="../../lib/css/header.css"
+    />
+    <link
+      rel="stylesheet"
+      href="../assets/demo.css"
+    />
+    <script src="../assets/theme.js"></script>
+    <style>
+      .iui-thumbnail {
+        background-image: url('../assets/image-test.jpg');
+      }
+
+      .demo-side-navigation {
+        background-color: var(--iui-color-background-1);
+        width: 72px;
+        height: 55px;
+        border-right: 1px solid var(--iui-color-background-5);
+        border-bottom: 1px solid var(--iui-color-background-5);
+        display: flex;
+        justify-content: center;
+        align-items: center;
+      }
+
+      .demo-side-navigation-icon {
+        width: 24px;
+        height: 24px;
+        fill: var(--iui-icons-color-actionable);
+      }
+
+      .iui-input {
+        border-radius: 22px;
+        width: 20vw;
+        background-color: var(--iui-color-background-2);
+      }
+
+      body {
+        background-color: var(--iui-color-background-2) !important;
+      }
+
+      section {
+        width: auto !important;
+      }
+    </style>
+  </head>
+  <body class="iui-body">
+    <theme-button></theme-button>
+    <h1>Header</h1>
+    <hr />
+
+    <h2>Default</h2>
+    <section id="demo-default">
+
+      <header class="iui-header">
+        <div class="iui-left">
+          <!-- Application Button -->
+          <button
+            class="iui-button iui-borderless iui-large iui-application"
+            type="button"
+            aria-label="Microstation"
+          >
+            <span class="iui-brand-logo">
+              <svg
+                viewBox="0 0 16 16"
+                xmlns="http://www.w3.org/2000/svg"
+                aria-hidden="true"
+                class="iui-icon"
+              >
+                <path
+                  d="m12.6 13.4c-1.2-1.5-2.1-3.1-2.4-5.5-2.7 3.9-4.6 4.2-5.7 2.4l-1.2 5.7h-2.2l3.5-14.1 1.8-.4c-.1.5-1.4 6.2.6 7 2 .7 4.6-8.5 4.6-8.5l2.2.4c-1.6 3.7-1.6 7.6 1.1 10.9l-2.3 2.1"
+                />
+              </svg>
+            </span>
+            <span class="iui-label">MicroStation</span>
+          </button>
+
+          <div class="iui-divider"></div>
+
+          <!-- Project Button -->
+          <button
+            class="iui-button iui-borderless iui-header-button iui-detailed"
+            type="button"
+            id="test-button-1"
+          >
+            <div class="iui-thumbnail-area">
+              <div class="iui-thumbnail"></div>
```

Comment:
Can `<img>` be used here?

Reply:
I think we'll have stretching issues if the image used is not a 1:1 square.